src/lib.rs

```diff
@@ -169,6 +169,7 @@ macro_rules! cmd {
             $crate::cmd($program, empty::<OsString>())
         }
     };
+    ( $program:expr, $( $arg:expr, )* ) => { cmd!($program $(, $arg )+ ) };
```

Comment:
Do you know if the `+` on the right side is equivalent to `*`? I know it's different on the left, but I guess both just mean "splat out all the matches" when you use them on the right?

Reply:
Yes it seems that it is. should I change?